app/src/main/java/org/mozilla/fenix/downloads/DownloadNotificationBottomSheetDialog.kt

```diff
@@ -35,9 +38,11 @@ class DownloadNotificationBottomSheetDialog(
             download_notification_title.text =
                 context.getString(R.string.mozac_feature_downloads_failed_notification_text2)
 
-            download_notification_icon.setImageDrawable(context.getDrawable(
-                mozilla.components.feature.downloads.R.drawable.mozac_feature_download_ic_download_failed
-            ))
+            download_notification_icon.setImageDrawable(
```

Comment:
nit: Since you have to fix a conflict here anyways...do you mind changing this to `setImageResource`? That should use `AppCompatResources` under the hood for the compat ImageView.